person.rb

```diff
@@ -1,15 +1,16 @@
 require './nameable'
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- Please `rentals` is both a getter and setter. that means it can be declared as both `attr_writer` and `attr_reader`
- Therefore move it to `attr_accessor`(Which is also a shortcut). [Open the link to see the example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)